### ISSUE TITLE
When isReverse is true,The scroll bar position is always at the top.

### ISSUE THREAD

**Issue by hengkx** [CassetteRocks/react-infinite-scroller#183]
**Describe the bug**
The scroll bar position is always at the top. should show current content

**To Reproduce**
[this Code Sandbox](https://codesandbox.io/s/9z11j4zjpo)

**Expected behavior**
When isReverse is true,loadMore change scroll bar position

**Screenshots**

**Device (please complete the following information):**
 - OS: windows
 - Browser chrome
 - Version latest

**Comment by SteveAquino** [comment]
I ran into this issue as well.  If you scroll to the top, the DOM keeps the scrollbar "stuck" at a position of 0. If you scroll slowly (eg, never reach the very top) then this issue doesn't happen.

I was able to get around this by setting a low threshold (40) and manually scrolling to just below the threshold (50).

```javascript
/**
   * Manually scroll to just below the threshold.
   */
  scrollToTop = () => {
    return this.scrollView && this.scrollView.scrollComponent.parentElement.scroll(0, 50);
  }

  /**
   * Passed as the `loadMore` prop.
   */
  loadMore = () => {
    if (this.state.refreshing) {
      return;
    }

    this.setState({ refreshing: true }, () =>
      this.props.fetchData().then(() => {
        this.scrollToTop();
        this.setState({ refreshing: false });
      })
    );
  }

  render() {
    return (
      <InfiniteScroll
        isReverse
        threshold={40}
        loadMore={this.loadMore}
        ref={ref => this.scrollView = ref}
      >
        { /* Render content */ }
      </InfiniteScroll>
    );
  }
```

**Comment by danbovey** [comment]
Released in version `1.2.3`!
